src/SFA.DAS.ApprenticeCommitments.Jobs.Functions/ApprenticeshipCommitmentsJobsHandler.cs

```diff
@@ -20,11 +23,31 @@ public async Task Handle(ApprenticeshipCreatedEvent message, IMessageHandlerCont
             }
             else
             {
-                await api.CreateApprentice(message.ToApprenticeshipCreated());
+                // create registration
```

Comment:
I'd prefer structural comments like this to be turned into method calls if they're necessary.  In this case I think we can safely remove the comment, as what is actually happening is "create apprentice", which is nicely embodied by the method call :)

Reply:
ok ill remove the comment - I though it adds clarity to what the call is actually doing